hooks/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+FOO=${IMAGE_NAME:=`whoami`/docker-alpine-s3fs:local}
+BAR=${SOURCE_BRANCH:=`git rev-parse --abbrev-ref HEAD`}
```

Comment:
You're correct that Sue and bar not used. It was done that way for syntactic cleanliness. The lifespan of those variables are minimal and not of any consequence to the build process but it did provide clarity in the Assembly of the image name and Source Branch variables.